packages/vega/src/index.ts

```diff
@@ -94,10 +94,9 @@ export const generateBasicAnnotations = async (
       onboardingMessages = scatterplotFactory(contextKey, vegaSpec, d3Data, visElement)
       break
 
-    default:
-      throw new Error(
-        `No onboarding for visualization type ${visType} available.`
-      )
```

Comment:
same here

Reply:
Added default message also 'No onboarding for visualization type ${visType} available. Please use visualization type 'GENERIC' to create your own onboardings.'